package-lock.json

```diff
@@ -11953,6 +11953,7 @@
           "resolved": "https://registry.npmjs.org/extend-shallow/-/extend-shallow-2.0.1.tgz",
           "integrity": "sha1-Ua99YUrZqfYQ6huvu5idaxxWiQ8=",
           "dev": true,
+          "optional": true,
```

Comment:
we should not have package-lock.json changes in PR

Reply:
fixed